python/lsst/ts/observing/utilities/logging.py

```diff
@@ -0,0 +1,143 @@
+# This file is part of ts_observing_utilities.
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+import sys
+
+BLACK, RED, GREEN, YELLOW, BLUE, MAGENTA, CYAN, WHITE = range(8)
+
+RESET_SEQ = '\033[0m'
+COLOR_SEQ = '\033[1;%dm'
+BOLD_SEQ = '\033[1m'
+
+COLORS = {
+    'WARNING': YELLOW,
+    'INFO': WHITE,
+    'DEBUG': BLUE,
+    'CRITICAL': RED,
+    'ERROR': RED
+}
+
+
+def get_logger(logger_name=None, use_color=True):
+    """
+    Return a logger with the "logger_name".
+
+    Parameters
+    ----------
+    logger_name : str, optional
+        The logger name to be used in different contexts.
+    use_color : bool, optional
+        Use colors on Stream Loggers.
+
+    Returns
+    ------
+    logging.Logger : the logger to be used.
+    """
+    message_format = " [%(levelname).1s %(asctime)s %(name)s] %(message)s"
+    date_format = "%Y-%m-%d %H:%M:%S"
+
+    _logger = logging.getLogger(logger_name)
+
+    if len(_logger.handlers) == 0:
+        formatter = MyLogFormatter(message_format, datefmt=date_format, use_colors=use_color)
+
+        handler = logging.StreamHandler(sys.stdout)
+        handler.setFormatter(formatter)
+
+        _logger.addHandler(handler)
+        _logger.setLevel(logging.DEBUG)
+
+    return _logger
+
+
+class MyLogFormatter(logging.Formatter):
```

Comment:
Consider renaming the class, how about `DecoratedLogger`? Also, I know I suggested naming the module `logging` but if you agree to rename the class, rename the module to match it, e.g. `decorated_logger`.